tests/integration/Installer/Installer/CoreInstallerTest.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace Ibexa\Tests\Integration\Installer\Installer;
+
+use EzSystems\PlatformInstallerBundle\Installer\CoreInstaller;
+use Ibexa\Tests\Integration\Installer\TestCase;
+use Symfony\Component\Console\Output\NullOutput;
+
+final class CoreInstallerTest extends TestCase
+{
+    /** @var \EzSystems\PlatformInstallerBundle\Installer\CoreInstaller */
+    private $installer;
```

Comment:
Is it possible to have:
```suggestion
    private CoreInstaller $installer;
```
instead? Or maybe even `\Ibexa\Bundle\RepositoryInstaller\Installer\Installer` if the code can work without specific impl hinted.

Reply:
I pushed Andrzej's suggestion before your comment though, @alongosz do you feel convinced? 😅

Reply:
Well, not anymore after f3e9afa :sweat_smile: Changing the requested service to `Installer` will break the test, because we only requested that `CoreInstaller` be exposed to test environment.

Reply:
From my point of view, I prefer to type-hint against a specific implementation here. We could type-hint against `Installer` and change the exposed service from `CoreInstaller` to `Installer` (interface), but to be fair service under test here is `CoreInstaller`.

We're testing that `CoreInstaller` can `importSchema` properly, not the interface.

Additionally, if we were to change it to `Installer` here, we would need to backport it to maintain consistency.